src/diffusers/models/attention.py

```diff
@@ -275,13 +275,13 @@ def __init__(
 
     def forward(
         self,
-        hidden_states,
-        attention_mask=None,
-        encoder_hidden_states=None,
-        encoder_attention_mask=None,
-        timestep=None,
-        cross_attention_kwargs=None,
-        class_labels=None,
+        hidden_states: Optional[FloatTensor],
```

Comment:
```suggestion
        hidden_states: torch.FloatTensor,
```

Reply:
restored namespace qualifier on all uses of *Tensor symbols